bench/BINARY_SIZE.md

```diff
@@ -18,7 +18,7 @@ Solana version: 2.1.0
 
 | Program | Binary Size | -                        |
 | ------- | ----------- | ------------------------ |
-| bench   | 1,041,928   | 🔴 **+250,920 (31.72%)** |
+| bench   | 1,106,136   | 🔴 **+315,128 (39.84%)** |
```

Comment:
It's crazy that this simple change adds ~60 KB to the program binary. I guess we could optimize it by not using the existing types such as `ProgramData` (which imports `bincode`) and interpreting the account data ourselves, but given we're planning to get rid of these instructions, it's probably fine to keep it as is and not complicate things further.

Reply:
Cool to see that you're tracking this. 60kb does sound way too much, but I agree with you here